pkg/kots/rego/kots-spec-opa-nonrendered.rego

```diff
@@ -553,6 +566,119 @@ lint[output] {
   }
 }
 
+# Check if repeatable ConfigOption has a template field defined
+lint[output] {
+  config_option := config_options[_]
+  item := config_option.item
+  item.repeatable
+  not item.template
+  field := concat(".", [config_option.field, "type"])
+  message := sprintf("Repeatable Config option \"%s\" is missing the template field", [string(item.name)])
+  output := {
+    "rule": "repeat-config-option-missing-template",
+    "type": "error",
+    "message": message,
+    "path": config_file_path,
+    "field": field,
+    "docIndex": config_data.docIndex
+  }
+}
+
+# Check if repeatable ConfigOption has at least one template
+lint[output] {
+  config_option := config_options[_]
+  item := config_option.item
+  item.repeatable
+  item.template == null
```

Comment:
how is this different from the previous rule?

Reply:
`not item.template` checks `does item have a field named 'template'?`

`item.template == null` checks `is the item.template value undefined?`
this check will not trigger if item does not have a field named `template`, which is why we need the previous rule as well.

Reply:
Removed after updating schema